update.php

```diff
@@ -319,12 +319,22 @@
 
 				if ( $config['pecl_extensions'] ) {
 					$install_extensions .= " \\\n\t\\\n";
-					$install_extensions .= array_reduce( $config['pecl_extensions'], function ( $command, $extension ) {
+
+					if ( version_compare( $version, '8.0beta1' ) >= 0 ) {
```

Comment:
Minor code readability suggestion (for here and other `version_compare()`s):
```suggestion
					if ( version_compare( $version, '8.0beta1' , '>=') === true ) {
```

Also `8.0beta1` is not a semver/official version number, so may not always work as expected, though in the context of this script should be fine as the `$version` is taken from the array.

Reply:
Thanks @jrfnl! I updated these `version_compare()` calls to include the operator. I also changed them to check for `> 7.4`, which doesn't use an "off" version number. 

I think initially I was using a version to match the way the [PHP Docker containers are named](https://hub.docker.com/_/php?tab=tags&page=5&name=8.0). As you mentioned, `$version` is taken from the array keys defined in the script. Since it's unlikely we will include specific minor versions of PHP in the configuration, comparing to just `7.4` should be sufficient to match PHP 8+.